spec/openapi.yaml

```diff
@@ -86,7 +82,7 @@ paths:
             This value should be in the format of `fieldname [ASC|DESC]`. For example, `installsrecent DESC`
           required: false
           schema:
-            type: string
+            $ref: "#/components/schemas/sort_whitelist"
```

Comment:
It seems like this component is missing from your PR. Is it a staged change that wasn't pushed maybe?